services/management/checks.go

```diff
@@ -131,3 +135,28 @@ func (s *ChecksAPIService) ChangeSecurityChecks(req *managementpb.ChangeSecurity
 
 	return &managementpb.ChangeSecurityChecksResponse{}, nil
 }
+
+// parses the check script and returns the docstring for the `check_context` function.
+func (s *ChecksAPIService) getCheckDescription(check check.Check) string {
+	// TODO there is similar code in checked service
+	// move this to a common package if possible.
+	var thread starlark.Thread
+	globals, err := starlark.ExecFile(&thread, "", check.Script, nil)
+	if err != nil {
+		s.l.Warnf("%s: failed to get check description, %s", check.Name, err)
+		return ""
+	}
+
+	fun, ok := globals["check_context"].(*starlark.Function)
+	if !ok {
+		s.l.Warnf("%s: no `check_context` function found", check.Name)
+		return ""
+	}
+	doc := strings.TrimSpace(fun.Doc())
+	if doc == "" {
+		s.l.Warnf("%s: `check_context` function should have docstring", check.Name)
```

Comment:
for now I am returning an empty string in case the docstring is not present, let me know if we expect and error here

Reply:
That's ok. We have it validated on SaaS side, so there we should handle it gracefully, and returning empty string is the best we can do